FEPs/FEP-0008-project-group-structure/README.md

```diff
@@ -0,0 +1,383 @@
+
+Working Group statements do not carry any special status over other community views. Maintainers should treat them as such when making decisions. If Maintainers cite a Working Group’s position as a rationale for a decision, the legitimacy of that decision derives from the quality and breadth of the group’s discussion—not from the mere existence of a Working Group statement.
+
+### Structure of a Working Group
+
+Each Working Group must have a designated **Chairperson** responsible for managing the group,
+defining its rules, and acting as the primary contact. No individual should chair more than one
+Working Group. The Chairperson must appoint at least one (ideally no more than two) **Co-Chairs** to
+ensure continuity during absences. Co-Chairs may serve in multiple groups. Collectively, the
+Chairperson and Co-Chairs are referred to as “the Chair”. Any decision regarding the Group that
+is not covered by the document is up for the Chair to decide.
+
+If a Chair is unable to fulfill their duties, the group members may request that the Maintainers
+appoint a replacement.
+
+#### Membership and Internal Structure
+
+The Chair determines the rules for membership, which must be non-discriminatory and allow anyone to
+request to join. These rules must be accessible to everyone and the desired communication link for the application must be clearly stated. Specific expertise may be required to become a member. Community members may belong
+to multiple Working Groups.
+
+New members of the working group are appointed by the Chair. Community member may request membership by
+contacting the Chair or asking over public channel. The Chair can accept or deny such request - in such
+case however the Chair should provide rationale for the decision, for example lack of required expertise. Not being a member does not however forbid anyone from taking part in the discussions.
+
+Existing members can be removed from the group by the Chair if they do not longer fulfill the
+requirements or the violate Code of Conduct [[2]](#code-of-conduct).
+
+Since official communication takes place on GitHub, all members must have a GitHub account.
+Membership is defined by the corresponding GitHub team: `@FreeCAD/<name>-working-group`.
+
+Groups may define additional roles, but the structure should be kept as simple as possible.
+
+### Communication Within a Group
+
+Groups may choose their preferred communication tools, but the chosen platform must allow free
+participation without barriers such as payment or special permissions. The method should be as
+transparent as possible, ideally allowing read-only access without requiring an account.
+
+Decisions (such as policy proposals, feature opinions, or issue feedback) must involve all group
+members and, where appropriate, input from the wider community to avoid echo chambers. The Chair is
```

Comment:
```suggestion
Decisions (such as policy proposals, feature opinions, or issue feedback) must involve all group
members and, where appropriate, input from the wider community to avoid missing on potentially important feedback. The Chair is
```